packages/iTwinUI-react/src/core/Table/Table.tsx

```diff
@@ -243,6 +244,24 @@ export type TableProps<
    * @default false
    */
   enableColumnReordering?: boolean;
+  /**
+   * Function that returns index of the row that you want to scroll to.
+   *
+   * It doesn't work with paginated tables.
+   * @example
+   * <Table
+   *   data={data}
+   *   scrollToRow={(rows) => rows.findIndex((row) => row.original === data[250])}
+   *   {...restProps}
+   * />
+   * @example
+   * <Table
+   *   scrollToRow={(rows) => rows.findIndex((row) => row.id === item.id)}
+   *   getRowId={(rowData) => row.original.id}
```

Comment:
so `getRowId` is not needed anymore, right?

Reply:
Yes, it is up to the users how they want to implement this.